### PR TITLE
TestCommon: Add DatabricksSchemaManager extensions

### PR DESCRIPTION
**Pull request by dstenroejl**
## Description

- [x] Bump NuGet packages to latest versions
- [x] Add DatabricksSchemaManager extensions

Related to:
https://app.zenhub.com/workspaces/mosaic-60a6105157304f00119be86e/issues/gh/energinet-datahub/team-mosaic/228

## Quality

- [x] Documentation is updated
- [x] Release notes are updated
- [x] Package version is updated
- [x] Public types and methods are documented
- [ ] Tests are implemented and executed locally

### CODE REVIEW

File: source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    /// <summary>
+    /// Expects a CSV file which was exported from Databricks.
+    /// It means the header must contain the column names and each row must contain the delta table values per column.
+    /// Delta table arrays must be parsed differently, but otherwise all values can be parsed from the CSV into strings.
+    /// All parsed rows are then inserted into a delta table.
+    /// </summary>
+    public static async Task InsertFromCsvFileAsync(
+        this DatabricksSchemaManager schemaManager,
+        string tableName,
+        Dictionary<string, (string DataType, bool IsNullable)> columnDefinition,
```
Comment (ebbeknudsen) [inline source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs:33]:
```suggestion
        Dictionary<string, (string DataType, bool IsNullable)> columnDefinitions,
```

File: source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+namespace Energinet.DataHub.Core.FunctionApp.TestCommon.Databricks;
+
+public static class DatabricksSchemaManagerExtensions
+{
+    public const string DeltaTableArrayOfStrings = "ARRAY<STRING>";
+
+    /// <summary>
+    /// Expects a CSV file which was exported from Databricks.
+    /// It means the header must contain the column names and each row must contain the delta table values per column.
```
Comment (ebbeknudsen) [inline source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs:26]:
```suggestion
    /// The header must contain the column names and each row must contain the delta table values per column.
```

File: source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            await schemaManager.InsertAsync(tableName, csvReader.HeaderRecord, rows).ConfigureAwait(false);
+        }
+    }
+
+    /// <summary>
+    /// Parse CSV column value into a delta table "insertable" value.
+    /// Only arrays require special handling; all other values can be inserted as "strings".
+    /// </summary>
+    private static string ParseColumnValue(
+        Dictionary<string, (string DataType, bool IsNullable)> columnDefinition,
```
Comment (ebbeknudsen) [inline source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs:66]:
```suggestion
        Dictionary<string, (string DataType, bool IsNullable)> columnDefinitions,
```

File: source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        var columnValue = csvReader.GetField(columnIndex);
+
+        if (columnDefinition[columnName].IsNullable && columnValue == string.Empty)
+        {
+            return "NULL";
+        }
+
+        if (columnDefinition[columnName].DataType == DeltaTableArrayOfStrings)
+        {
+            var arrayContent = columnValue!
```
Comment (ebbeknudsen) [inline source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs:80]: Do we need to check if columnValue can be `null` as well as `string.Empty`?

**Review by ebbeknudsen — DISMISSED** [review]

File: source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs (hunk 4 above)
Comment (dstenroejl) [reply source/TestCommon/source/FunctionApp.TestCommon/Databricks/DatabricksSchemaManagerExtensions.cs:80]: No

**Review by ebbeknudsen — APPROVED** [review]

**Review by ebbeknudsen — APPROVED** [review]
LGTM

**Pull request merged**